File: LongChallenge_2018_12_CHFINTRO.java
// code_report Solution
// https://youtu.be/aWBkxc__VDY

import java.util.Scanner;

public class Main {
     public static void main(String []args){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int r = sc.nextInt();
        while (n-- != 0) {
            int x = sc.nextInt();
            System.out.println(x < r ? "Bad boi" : "Good boi");
            System.out.flush();
        }
     }
}

File: LongChallenge_2018_12_CHFINTRO.cpp
// code_report Solution
// https://youtu.be/aWBkxc__VDY

#include <iostream>
using namespace std;

int main() {
   int n, r; cin >> n >> r;
   while (n--) {
      int x; cin >> x;
      cout << (x < r ? "Bad boi" : "Good boi") << endl;
   }
   return 0;
}
